src/utils/getLocalESLint.js

```diff
@@ -3,8 +3,19 @@ const findUp = require('find-up');
 
 const getLocalESLint = config => {
   const nodeModulesPath = findUp.sync('node_modules', { cwd: config.rootDir });
-  // eslint-disable-next-line import/no-dynamic-require, global-require
-  return require(path.resolve(nodeModulesPath, 'eslint'));
+  try {
+    // eslint-disable-next-line import/no-dynamic-require, global-require
+    return require(path.resolve(nodeModulesPath, 'eslint'));
```

Comment:
what about just using https://www.npmjs.com/package/resolve-from here?

Replace the whole `getLocalESLint` with just a `resolveFrom(config.rootDir, 'eslint');`.

I think that should be enough (and it means `findUp` should not be needed)

Reply:
I’d prefer not to use that package due to its aggressive dropping of support of older platforms. Could `resolve` work here?

Reply:
seems like it takes `basedir`, so `resolve.sync('eslint', { basedir: config.rootDir })` should in theory do the trick as well